packages/web-api/src/types/response/MigrationExchangeResponse.ts

```diff
@@ -17,9 +17,6 @@ export type MigrationExchangeResponse = WebAPICallResult & {
   ok?:               boolean;
   provided?:         string;
   team_id?:          string;
-  user_id_map?:      UseridMap;
+  user_id_map?:      { [key: string]: string };
   warning?:          string;
 };
-
-export interface UseridMap {
```

Comment:
⚠️ **BREAKING CHANGE**

If anyone was using that interface. Though given it was not very useful and cast was needed anyway, removing usage would be enough to fix it.

Reply:
How exactly is this a breaking change? Because the values within `user_id_map` were typed to `any` before, and now would be a `string`?

Reply:
I don’t consider this a breaking change. The auto-generated types here have been changed based on quicktype outputs over time. We've been trying to maintain backward compatibility as much as possible, but when an existing type is incomplete and/or invalid, changes like this can happen.

Reply:
Agreed. Perhaps in the future, if we start paying closer attention to the response types, and start augmenting them with manual types, we could employ a more stringent backwards compatibility policy for the response types.

Reply:
It's technically a breaking change regarding to type definitions as:
1. **Interface `UseridMap` is exported**. It's also included in published `dist/` files. So it's part of public API surface
1. **Interface `UseridMap` gets removed**. Which is a breaking change in case someone was importing it & using it.
1. **Type definition is narrower**. Before it was very open ([any non null type](https://typescript-eslint.io/rules/no-empty-interface/)). Now it's an [indexed type](https://www.typescriptlang.org/docs/handbook/2/objects.html#index-signatures) which is a stricter type. So it's also non backwards compatible. 

But given the type needed a cast anyways and cast was needed it's probably not enough for a new major version bump due to this kind of small breaking change. And more if auto-generated types are following a good-enough approach. 

I'm happy with whatever you decide, reasons sound fair enough to me :)